docs/en/current/settings/export.md

```diff
@@ -0,0 +1,22 @@
+# Export Settings 
+
+K-9 provides the ability to import and export your settings. This is useful when changing/adding a device.
+
+The settings are exported as a file, stored in a location of your choice on the device.
```

Comment:
```suggestion
The settings are exported as a file, stored in a location of your choice.
```

We're using Android's Storage Access Framework. This supports remote storage, too.